src/Lazy.ts

```diff
@@ -82,6 +82,16 @@ class Lazy<T extends AnySchema, TContext = ContextOf<T>>
   describe() {
     return null as any;
   }
+
+  isValid(value: any, options?: ValidateOptions<TContext>) {
+    return this._resolve(value, options).isValid(value, options);
+  }
+  isValidSync(value: any, options?: ValidateOptions<TContext>) {
+    return this._resolve(value, options).isValidSync(value, options);
+  }
+  isType(value: any) {
+    return this._resolve(value).isType(value);
+  }
```

Comment:
hmm without the resolve options this will likely return the wrong thing, or be incorrect, but adding options here makes the method signature different than the `schema.isType`...maybe better not to add this one?

Reply:
Hmm, I thought I was using it successfully in my own project but apparently not, so I'm not sure. I went ahead and removed it.